DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageViewRecorder.swift

```diff
@@ -152,7 +154,7 @@ internal struct UIImageViewWireframesBuilder: NodeWireframesBuilder {
                 builder.createPlaceholderWireframe(
                     id: imageWireframeID,
                     frame: clipsToBounds ? relativeIntersectedRect : contentFrame,
-                    label: "Content Image"
+                    label: imagePrivacyLevel == .maskContent ? "Content Image" : "Image"
```

Comment:
@jonathanmos maybe long term it's worth having a single placeholder like `Masked Image` or just `Image` for everything